### ISSUE TITLE
Modify Email Signup Form

### ISSUE THREAD

**Issue by aparkening** [alistapart/AListApart#330]
Modify email signup at http://alistapart.com/email-signup to reflect new lists.
## Current Options:
You can get an email whenever we have something fresh:
- Articles (Our deepest dives. Published every other Tuesday.)
- Columns (Smart perspectives from community leaders. About once a week.)
- News and Events (ALA On Air invitations and other occasional updates.)
- Blog posts (ALA updates and hands-on advice. Roughly twice per week.)

Or, receive a roundup of all our recent content whenever we publish a new issue:
- Digest (Sent every other Tuesday.)

## Updated Options:
- Something like "Get an email whenever we have something fresh"

Or, receive a roundup of all our recent content whenever we publish a new issue:
- Digest (Sent every other Tuesday.)

@litherland: can you provide better wording for the feed of everything we publish?  The "Get an email whenever we have something fresh" option above.

**Comment by litherland** [comment]
Proposed changes:

>Receive ALA in Your Email

→ 

>Get ALA via Email

(The current `h1` is fine, if a bit formal.  What’s proposed is just a little simpler,  pithier.  Also,  “receive” is repeated immediately afterward; this changes things up.)

>You can get an email whenever we have something fresh:

→ 

>Get an email when we publish something fresh!

(A little tighter, a little more enthusiastic.)

>Or, receive a roundup of all our recent content whenever we publish a new issue:

→ 

>Or, get a roundup of all our recent content when we publish a new issue.

(A little tighter. Note the full stop instead of the colon.)

**Comment by aparkening** [comment]
Just in case previous formatting was hard to parse, the new wording is:
- Get an email when we publish something fresh!
- Or, get a roundup of all our recent content when we publish a new issue.
